arm-oraclelinux-wls-cluster/src/main/arm/nestedtemplates/aadNestedTemplate.json

```diff
@@ -144,11 +162,36 @@
                 }
             }
         },
+        {
+            "type": "Microsoft.Compute/virtualMachines/extensions",
+            "name": "[concat(variables('const_managedVMPrefix'), add(copyIndex(), 1),'/newuserscript')]",
+            "apiVersion": "2019-07-01",
+            "location": "[parameters('location')]",
+            "copy": {
+                "name": "managedVirtualMachineExtensionLoop",
+                "count": "[add(parameters('numberOfInstances'), -1)]"
+            },
+            "properties": {
+                "publisher": "Microsoft.Azure.Extensions",
+                "type": "CustomScript",
+                "typeHandlerVersion": "2.0",
+                "autoUpgradeMinorVersion": true,
+                "settings": {
+                    "fileUris": [
+                        "[uri(parameters('_artifactsLocationAADTemplate'), concat('../scripts/', variables('name_scriptAADConfiguration'), parameters('_artifactsLocationSasToken')))]"
+                    ],
+                    "commandToExecute": "[concat('sh',' ',variables('name_scriptAADConfiguration'),' ', variables('const_aadParameters'),' ', add(copyIndex(),1))]"
+                }
+            }
+        },
```

Comment:
Update managed VMs first, we will restart managed server by running WLST script in admin VM to make the changes work.